core/server/data/migrations/versions/2.2/2-add-mobiledoc-revisions-table.js

```diff
@@ -0,0 +1,35 @@
+const common = require('../../../../lib/common');
```

Comment:
Why is the migration script still here? Weird

Reply:
Ah, its because the migration number for this changed to `5-` instead of `2-` in the final merge on master, and this one is still using the old filename 🤦‍♂️

Reply:
Deleting this file